### ISSUE TITLE
Failure building libminpack during wheel build

### ISSUE THREAD

**Issue by bennahugo** [lofar-astron/PyBDSF#113]
This looks like a bug introduced in the Nov 19 release on pypi. It fails building libminpack (I've already installed both 18.04 libminpack1 minpack-dev packages along with the packages on the readme). I can reproduce this in a new python2.7 virtualenv with numpy preinstalled (which should probably be bundled as a PEP 518 setup requirement to ensure bdsf can be installed as a requirement in a clean environment).

I've additionally upgraded pip setuptools and wheel to current versions:
```
(venv2) > $ python2.7 -c "import wheel; print wheel.__version__"             
0.33.6
(venv2) > $ python2.7 -c "import pip; print pip.__version__"                 
19.3.1
(venv2) > $ python2.7 -c "import setuptools; print setuptools.__version__"   
41.6.0
```
This does not look like a gcc error. Any ideas on what the missing dependencies are?

```
  copying bdsf/mylogger.py -> build/lib.linux-x86_64-2.7/bdsf
  copying bdsf/tc.py -> build/lib.linux-x86_64-2.7/bdsf
  creating build/lib.linux-x86_64-2.7/bdsf/nat
  copying bdsf/nat/__init__.py -> build/lib.linux-x86_64-2.7/bdsf/nat
  running build_ext
  running mbuild
  Building libminpack
  error: [Errno 2] No such file or directory: '/tmp/pip-build-lXDlWC/bdsf/src/minpack'
  
  ----------------------------------------
[91m  Failed building wheel for bdsf
[0m  Running setup.py clean for bdsf
  Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-lXDlWC/bdsf/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" clean --all:
  running clean
  running mclean
  Removing libminpack.a and object files
  error: [Errno 2] No such file or directory: '/tmp/pip-build-lXDlWC/bdsf/src/minpack'
  
  ----------------------------------------
[91m  Failed cleaning build dir for bdsf
[0mFailed to build bdsf
Installing collected packages: backports.shutil-get-terminal-size, bdsf
  Running setup.py install for bdsf: started

    Running setup.py install for bdsf: finished with status 'error'
    Complete output from command /usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-lXDlWC/bdsf/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-ZYepAf-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running config_cc
    unifing config_cc, config, build_clib, build_ext, build commands --compiler options
    running config_fc
    unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
    running build_src
    build_src
    building extension "bdsf._pytesselate" sources
    f2py options: ['']
      adding 'build/src.linux-x86_64-2.7/build/src.linux-x86_64-2.7/bdsf/fortranobject.c' to sources.
      adding 'build/src.linux-x86_64-2.7/build/src.linux-x86_64-2.7/bdsf' to include_dirs.
    building extension "bdsf._cbdsm" sources
    building extension "bdsf.nat.natgridmodule" sources
    build_src: building npy-pkg config files
    running build_py
    running build_ext
    running mbuild
    Building libminpack
    error: [Errno 2] No such file or directory: '/tmp/pip-build-lXDlWC/bdsf/src/minpack'
    
    ----------------------------------------
[91mCommand "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-lXDlWC/bdsf/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-ZYepAf-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-lXDlWC/bdsf/
[0mThe command '/bin/sh -c pip install bdsf' returned a non-zero code: 1
Build step 'Execute shell' marked build as failure
```

**Comment by tammojan** [comment]
Thanks for reporting, I'll try to look into this shortly.

**Comment by tammojan** [comment]
Hi @bennahugo, I cannot reproduce this. This may be due to `gfortran` not being available.

**Comment by bennahugo** [comment]
Hi thanks for checking. However, gfortran is installed. I'm running into
this issue in a minimal installation  of ddfacet with a docker file that
used to work for the earlier version of bdsf. I will put together a minimal
dockerfile to reproduce the issue by early next week.

**Comment by tammojan** [comment]
In the meantime, I've released 1.9.2 which is also released as binary wheels. Could you try if this works for you

**Comment by bennahugo** [comment]
@tammojan the binary wheel works in a python 3 environment so I'm going to close this ticket as python 2 is now dead. Thanks
